packages/manager/apps/pci-billing/src/components/consumption/HourlyConsumption.component.tsx

```diff
@@ -173,7 +181,7 @@ export default function HourlyConsumption({
   ];
 
   const renderAccordion = (
-    key: string,
+    key: keyof typeof consumption.totals.hourly,
```

Comment:
This generates a type error in line 209, in the usage of `renderAccordion`, because key is a string in the given parameter